capacitor.config.json

```diff
@@ -9,5 +9,9 @@
       "launchShowDuration": 0
     }
   },
-  "cordova": {}
+  "cordova": {},
+  "server": {
+    "url": "http://192.168.1.105:8100",
+    "cleartext": true
```

Comment:
@NickPhura is this gonna be a problem?  I really don't want to merge in anything that will break it for you guys

Reply:
Not sure, I dont think so?

I manually allowed cleartext in the android manifest at one point to fix a problem.

Dont really know much about it other than that